src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/cumulative-metrics-transition-guide.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Transition guide for OpenTelemetry cumulative sum metrics
+metaDescription: 'A transition guide about a change in how New Relic handles OpenTelemetry cumulative sum metrics, and how to update your queries and alerts for that change.'
+redirects: 
+---
+
+Our OTLP metric pipeline is being upgraded to better support how we handle [cumulative metrics](/docs/data-apis/understand-data/metric-data/cumulative-metrics). As part of this upgrade we're transitioning the underlying [metric data type](/docs/data-apis/understand-data/metric-data/metric-data-type/#metric-types) for [monotonic cumulative sums](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#sums) from a `gauge` type to the `cumulativeCount` type. The `cumulativeCount` type allows you to query both the cumulative and delta value associated with reported data. 
+
+Starting March 14, 2023, new New Relic customers will use the new `cumulativeCount` type, but if you were a New Relic customer before that date, you won't have access to that metric type until June 13. On that date, the use of our `gauge` metric type for cumulative sums will be deprecated and reach end-of-life (EoL), and your cumulative sums will be stored as the `cumulativeCount` type. 
```

Comment:
I stumbled a bit over the "new New Relic" phrase. How about revising like this: Starting March 14, 2023, customers who are getting started with New Relic...."

Also, will New Relic be converting my existing monotonic cumulative sum data in NRDB, so that if I want to query historical sums, I'll need to modify my queries for cumulativeCount? Or, is this just for all future sums?

Reply:
Existing monotonic cumulative sum data that was ingested prior to the EoL (i.e. stored as a `gauge` type in NRDB) will not be converted after the fact. Those data points will be retained per the defined retention duration in the account, so they will continue to be available for that period. To view both datatypes in a single NRQL view, something like the following could be used:
**cumulativeCount Query 1**
```
FROM Metric SELECT rate(sum(<metricName>), 1 minute) as 'delta rate', latest(<metricName>['cumulative']) as 'cumulative' WHERE <metricName>[type] = 'cumulativeCount'  TIMESERIES 
```
**gauge Query 2**
```
FROM Metric SELECT derivative(<metricName>, 1 minute) as 'gauge derivative', latest(<metricName>) as 'latest' WHERE <metricName>[type] = 'gauge' TIMESERIES 
```

I've tested these queries across the metric type boundary and they return as you would expect. The examples are bit verbose so I was hesitated to include them anywhere.